about.ftd

```diff
@@ -3,6 +3,7 @@
 -- import: fifthtry.github.io/admint.io/lib
 
 -- lib.page:
+active-page: about/
```

Comment:
@priyanka9634 can't we handle it without adding a new property in page component?

Reply:
To achieve this fixes required in sitemap and toc processor.